x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/form.test.tsx

```diff
@@ -474,6 +474,31 @@ describe('Event filter form', () => {
       rerender();
       expect(renderResult.findByTestId('duplicate-fields-warning-message')).not.toBeNull();
     });
+
+    it('should not show warning callout when wildcard is used with the "MATCHES" operator', async () => {
+      formProps.item.entries = [
+        {
+          field: 'event.category',
+          operator: 'included',
+          type: 'wildcard',
+          value: 'valuewithwildcard*',
+        },
+      ];
+      rerender();
+      expect(renderResult.queryByTestId('wildcardWithWrongOperatorCallout')).toBeNull();
+    });
+    it('should show warning callout when wildcard is used with the "IS" operator', async () => {
+      formProps.item.entries = [
+        {
+          field: 'event.category',
+          operator: 'included',
+          type: 'match',
+          value: 'valuewithwildcard*',
+        },
+      ];
+      rerender();
+      expect(renderResult.findByTestId('wildcardWithWrongOperatorCallout')).not.toBeNull();
```

Comment:
This is not doing what you think it is doing - because you are using `findBy**` the assertion (`.not.toBeNull()`) will always be true. Change it to `getByTestId()` if the expectation is for the component to be in the DOM -or- add `await` behind `expect(...`